build.assets/build-package.sh

```diff
@@ -259,14 +260,14 @@ pushd "$(mktemp -d)"
 PACKAGE_TEMPDIR=$(pwd)
 # automatically clean up on exit
 trap 'rm -rf ${PACKAGE_TEMPDIR}' EXIT
-mkdir -p "${PACKAGE_TEMPDIR}"/buildroot
+mkdir -p "${PACKAGE_TEMPDIR}/buildroot"
 
 # implement a rudimentary download cache for repeat builds on the same host
 mkdir -p "${TARBALL_DIRECTORY}"
-if [ ! -f "${TARBALL_DIRECTORY}"/"${TARBALL_FILENAME}" ]; then
+if [ ! -f "${TARBALL_DIRECTORY}/${TARBALL_FILENAME}" ]; then
     if [[ "${DOWNLOAD_IF_NEEDED}" == "true" ]]; then
         echo "Downloading ${URL} to ${TARBALL_DIRECTORY}"
-        curl -sL "${URL}" -o "${TARBALL_DIRECTORY}"/"${TARBALL_FILENAME}"
+        curl -sL "${URL}" -o "${TARBALL_DIRECTORY}${TARBALL_FILENAME}"
```

Comment:
Whoops, I removed the slash here.

```suggestion
        curl -sL "${URL}" -o "${TARBALL_DIRECTORY}/${TARBALL_FILENAME}"
```